fog/api/src/lib.rs

```diff
@@ -68,15 +68,15 @@ impl EnclaveGrpcChannel for ledger_grpc::FogKeyImageApiClient {
         &mut self,
         msg: &attest::AuthMessage,
         call_option: CallOption,
-    ) -> Result<(Option<Metadata>, attest::AuthMessage, Option<Metadata>), grpcio::Error> {
-        <Self>::auth_full(self, msg, call_option)
+    ) -> GrpcResult<(Metadata, attest::AuthMessage, Metadata)> {
+        <Self>::auth_async_opt(self, msg, call_option)?.receive_sync()
     }
     fn enclave_request(
         &mut self,
         msg: &attest::Message,
         call_option: CallOption,
-    ) -> Result<(Option<Metadata>, attest::Message, Option<Metadata>), grpcio::Error> {
-        <Self>::check_key_images_full(self, msg, call_option)
+    ) -> GrpcResult<(Metadata, attest::Message, Metadata)> {
+        <Self>::check_key_images_async_opt(self, msg, call_option)?.receive_sync()
```

Comment:
Again here, this looks like a case of "should be fine, but we should acceptance test it before the release is made"

Reply:
The `_full` suffix was our proposal, implemented only in our fork. The functionality has been implemented in the published crate with the `_async_opt` suffix.